addons/payment_authorize/tests/test_authorize.py

```diff
@@ -188,7 +188,7 @@ def test_20_authorize_form_management(self):
         self.assertEqual(tx.state, 'cancel', 'Authorize: erroneous validation did not put tx into error state')
 
 
-@odoo.tests.tagged('post_install', '-at_install', '-standard', 'external')
```

Comment:
is it intended?

Reply:
Yes, first the test has never been launched because it doesn't have any credential. And then this test cannot work on runbot, because as it is a server to server communication, and there are more than one database on runbot url the request sent  by the authorize server won't reach the good database.